docs/design/features/globalization-icu.md

```diff
@@ -0,0 +1,131 @@
+# .NET Globalization ICU on all runtimes
+
+Author: [Santiago Fernandez Madero](https://github.com/safern)
+
+In .NET Core our globalization APIs behavior used different underlying libraries, in Unix, we used [International Components for Unicode (ICU)](http://site.icu-project.org/home) and in Windows, we used [National Language Support (NLS)](https://docs.microsoft.com/en-us/windows/win32/intl/national-language-support). This caused some behavior differences in a handful number of Globalization APIs:
+
+- Cultures and culture data
+- String casing
+- String sorting and searching
+- Sort keys
+- String Normalization
+- Internationalized Domain Names (IDN) support
+- Time Zone display name on Linux
+
+## ICU on Windows
+
+Windows 10 May 2019 Update and later versions started including [icu.dll](https://docs.microsoft.com/en-us/windows/win32/intl/international-components-for-unicode--icu-) as part of the OS, so in .NET 5.0 we changed the default behavior for Globalization APIs when running in Windows, which tries to load `icu.dll` and if we can load it, we use ICU as the underlying native APIs, if we fail to load it, we use the legacy behavior (NLS), this will be the case for older versions of Windows.
+
+*Note: CurrentCulture, CurrentUICulture and CurrentRegion still use Windows OS APIs to honor user settings*
+
+### Using NLS instead of ICU
+
+Given that this is a change in the Globalization behavior, we added support for disabling this feature, so that applications can use legacy behavior which relies on: [National Language Support (NLS)](https://docs.microsoft.com/en-us/windows/win32/intl/national-language-support).
+
+Applications can enable NLS mode by either of the following:
```

Comment:
```suggestion
Using ICU instead of NLS may result in behavioral differences with some globalization-related operations. To revert back to using NLS, a developer can opt-out of the ICU implementation. Applications can enable NLS mode by either of the following:
```